_includes/_annotation_loader_tip.rst.inc

```diff
@@ -1,7 +1,7 @@
 .. note::
 
     In order to use the annotation loader, you should have installed the
-    ``doctrine/annotations`` and ``doctrine/cache`` packages with Composer.
+    ``sensio/framework-extra-bundle`` and ``doctrine/cache`` packages with Composer.
```

Comment:
I am not sure where we include this file, but is it needed in that context to have the SensioFrameworkExtraBundle or are these cases where `doctrine/annotations` is actually fine?

Reply:
We include this file in:

* /components/routing.rst
* /components/serializer.rst
* /components/validator/resources.rst

In 2.7, 2.8 and 3.4 we showed this recommendation:

```rst
.. note::

    In order to use the annotation loader, you should have installed the
    ``doctrine/annotations`` and ``doctrine/cache`` packages with Composer.
```

So let's keep it unchanged.